### PR TITLE
Add HTTP V2 test

### PR DESCRIPTION
**Pull request by CoderHam**

### CODE REVIEW

File: qa/L0_http_v2/test.sh [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    echo -e "\n***\n*** HTTP V2 Client Build Failed\n***"
+    RET=1
+fi
+
+DATADIR=`pwd`/models
+SERVER=/opt/tensorrtserver/bin/trtserver
+SERVER_ARGS="--model-repository=$DATADIR"
+source ../common/util.sh
+
+run_server_nowait
```
Comment (deadeyegoodwin) [inline qa/L0_http_v2/test.sh]: Just use run_server and then you don't need the sleep

Comment (CoderHam) [reply qa/L0_http_v2/test.sh]: **run_server** calls **wait_for_server_ready** which repeatedly curls the old HTTP url ->  `localhost:8000/api/health/ready` till it gets a success code 200 and hence would get stuck here when using HTTP V2

Comment (deadeyegoodwin) [reply qa/L0_http_v2/test.sh]: Can you put that comment in the script so we know why you are using it

Comment (CoderHam) [reply qa/L0_http_v2/test.sh]: Sure

**Pull request merged**